dataquality/core/init.py

```diff
@@ -191,3 +214,12 @@ def init(
         )
     # Reset all config variables
     dataquality.get_data_logger().logger_config.reset(factory=True)
+    if not run_created:
+        _set_labels_for_existing_run(project_name, run_name)
+
+
+def delete_run(project_name: str, run_name: str) -> None:
+    """Deletes a run from Galileo"""
+    pid, rid = api_client._get_project_run_id(project_name, run_name)
+    api_client.delete_run(pid, rid)
+    print(f"🗑 Run {project_name}/{run_name} deleted")
```

Comment:
Would this be nicer?

```suggestion
    print(f"🗑 Run {run_name} in project {project_name} deleted.")
```